src/yang/ietf-bfd-types.yang

```diff
@@ -115,25 +126,40 @@ module ietf-bfd-types {
   identity path-ip-sh {
     base path-type;
     description "BFD on IP single hop as described in RFC 5881.";
```

Comment:
Same as above "BFD on IP single hop." And similarly for all the comments below.

Reply:
So again this is because you find the description and reference information redundant? There was this email from Rob Shakir who was advocating for more info in descriptions:
"For the descriptions, there are tools that use these to build documentation automatically for a YANG model. We have found that such documentation is particularly useful for engineers that are actually consuming the models to generate config."

Reply:
We need to have better description, but not redundant information.